src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -110,6 +102,10 @@ public function __clone()
      */
     public function boot()
     {
+        if ($this->debug) {
+            $this->startTime = microtime(true);
```

Comment:
shouldn't we do this only when we actually do something (i.e. when not booted, or when the booting again without a request stack size) ?